### ISSUE TITLE
inability to delete contacts is a privacy issue

### ISSUE THREAD

**Issue by dbquarrel** [wireapp/wire-ios#4163]
## Checklist (please check before submitting)

I actually submitted this a couple of years ago and was arrogantly brushed off.

## The problem

The user cannot delete contacts. This is a basic feature available on every platform including email, since the dawn of computing. The only package on the planet that does not allow you to delete contacts is Wire. No good reason has ever been supplied for this and it appears to be that Wire wants to track contacts at the server and maintain this information on its users.

This is important because you may have:

1. added someone by accident, now you are forever tied to this person until you die
2. have some personal issue with someone and no longer want any contact with them 
3. someone like a fired employee, you may never wish to see on your contacts again
4. an ex-wife or ex-husband, potentially someone who is an abuser, seeing them in your contacts is something that is upsetting

For whatever reason you decide to cut ties with a person or organization, that you ever knew them is your own personal business. In this day and age when you can be forced to open your phone and display contacts, you may now be connected to people that were acceptable to you in the past but not now.

Wire, by not allowing you to delete contacts, is essentially keeping a permanent log of whom you have ever spoken with over your life. 

That, is what we call a major privacy issue and it needs to be addressed and not brushed off with attitude.

And no, blocking the contact is NOT the same as deleting them. This is why every other major messaging platform has separate functions for delete and block. You may wish to block someone and not delete them, you may wish to delete someone and not block them, you may wish to delete AND block them. This should be an absolute triviality to implement, and because Wire will not implement this while others do, at least a solid reason should be given to your users about why you are permanently tracking their contacts so that users can understand and make an informed decision about using this software.

## Environment

Every version of wire on every platform.

## Details

Add a contact. 

There is no delete function.

**Comment by cardassian-tailor** [comment]
I agree 100%. I too either submitted this same bug or commented on  yours, cant recall .. and I cant believe it hasn't been looked at.

**Comment by mHatsune** [comment]
https://github.com/wireapp/wire/issues/256
